### PR TITLE
OSDEV-946 Add error handling for RowSectorSerializer and RowFacilityTypeSerializer

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV-946](https://opensupplyhub.atlassian.net/browse/OSDEV-946)
Add error handling for `RowSectorSerializer` and `RowFacilityTypeSerializer` in the `contricleaner`

### CODE REVIEW

File: src/django/contricleaner/lib/serializers/row_serializers/row_composite_serializer.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def get_validated_row(self, raw_row: dict):
         }
 
         for res_key, res_value in res.items():
-            if res_key in standard_fields:
+            if res_key in standard_fields or res_key == "errors":
```
[the remark this answers is not in the document]
Comment (mazursasha1990) [reply src/django/contricleaner/lib/serializers/row_serializers/row_composite_serializer.py]: Done

File: src/django/contricleaner/lib/helpers/is_invalid_type.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+def is_invalid_type(value):
```
[the remark this answers is not in the document]
Comment (mazursasha1990) [reply src/django/contricleaner/lib/helpers/is_invalid_type.py]: Done

**Review by vladsha-dev — APPROVED** [review]
LGTM

File: src/django/contricleaner/lib/helpers/is_valid_type.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+def is_valid_type(value):
```
Comment (vladsha-dev) [inline src/django/contricleaner/lib/helpers/is_valid_type.py:1]: Could you please add type hints?

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: src/django/contricleaner/lib/helpers/is_valid_type.py (hunk 3 above)
Comment (mazursasha1990) [reply src/django/contricleaner/lib/helpers/is_valid_type.py]: Done

Comment (vladsha-dev) [reply src/django/contricleaner/lib/helpers/is_valid_type.py]: Do we have type `any` in Python?
I think you should use https://docs.python.org/3/library/typing.html#the-any-type

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by killev — APPROVED** [review]
Approved.

**Pull request merged**